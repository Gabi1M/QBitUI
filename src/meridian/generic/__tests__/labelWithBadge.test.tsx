import React from 'react';

import { render } from '@testing-library/react';

import LabelWithBadge from '../labelWithBadge';

describe('LabelWithBadge', () => {
    it('should render as expected', () => {
        const result = render(<LabelWithBadge label='Something' text='Some text' color='#FFFFF' />);
        expect(result.asFragment()).toMatchSnapshot();
    });
});
