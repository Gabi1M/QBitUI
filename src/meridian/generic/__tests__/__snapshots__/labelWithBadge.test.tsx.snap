// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LabelWithBadge should render as expected 1`] = `
<DocumentFragment>
  .emotion-2 {
  font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  -webkit-tap-highlight-color: transparent;
  color: inherit;
  font-size: inherit;
  line-height: 1.55;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-family: Greycliff CF,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  font-weight: 700;
}

.emotion-2:focus {
  outline-offset: 2px;
  outline: 2px solid #339af0;
}

.emotion-2:focus:not(:focus-visible) {
  outline: none;
}

.emotion-4 {
  -webkit-tap-highlight-color: transparent;
  font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  font-size: 11px;
  height: 20px;
  line-height: 18px;
  -webkit-text-decoration: none;
  text-decoration: none;
  padding: 0 10.666666666666666px;
  box-sizing: border-box;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  width: auto;
  text-transform: uppercase;
  border-radius: 32px;
  font-weight: 700;
  letter-spacing: 0.25px;
  cursor: default;
  text-overflow: ellipsis;
  overflow: hidden;
  background: rgba(231, 245, 255, 1);
  color: #228be6;
  border: 1px solid transparent;
  background-color: #FFFFF;
  margin-right: 5px;
}

.emotion-4:focus {
  outline-offset: 2px;
  outline: 2px solid #339af0;
}

.emotion-4:focus:not(:focus-visible) {
  outline: none;
}

.emotion-5 {
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
}

<div
    class="emotion-0"
  >
    <div
      class="emotion-1 emotion-2"
    >
      Something
    </div>
    <div
      class="emotion-3 emotion-4"
    >
      <span
        class="emotion-5 emotion-6"
      >
        Some text
      </span>
    </div>
  </div>
</DocumentFragment>
`;
